astropy/units/quantity.py

```diff
@@ -1736,30 +1736,86 @@ def _set_unit(self, unit):
         super()._set_unit(unit)
 
 
-def isclose(a, b, rtol=1.e-5, atol=None, **kwargs):
+def isclose(a, b, rtol=1.e-5, atol=None, equal_nan=False, **kwargs):
     """
+    Returns a boolean array where two arrays are element-wise equal
```

Comment:
Similarly, maybe
```
Whether elements of two arrays are equal within a tolerance
```
I think the numpy docstring is not very good...

But absolutely fine to stick with what you have!

Reply:
I'm wondering if this change could potentially lead to some ambiguity (like making it seem like `isclose` is doing what `allclose` does), so I'd prefer something closer to the original from NumPy.

Reply:
No worries, just do what you think is best.